src/main/kotlin/nl/info/zac/admin/ZaaktypeCmmnConfigurationBeheerService.kt

```diff
@@ -224,14 +225,9 @@ class ZaaktypeCmmnConfigurationBeheerService @Inject constructor(
             afrondenMail = previousZaaktypeCmmnConfiguration.afrondenMail
             productaanvraagtype = previousZaaktypeCmmnConfiguration.productaanvraagtype
             domein = previousZaaktypeCmmnConfiguration.domein
-            isSmartDocumentsIngeschakeld = previousZaaktypeCmmnConfiguration.isSmartDocumentsIngeschakeld
+            smartDocumentsIngeschakeld = previousZaaktypeCmmnConfiguration.smartDocumentsIngeschakeld
```

Comment:
Why rename this to something that does not follow the usual convention for boolean returning functions/methods?